packages/transaction-controller/CHANGELOG.md

```diff
@@ -6,6 +6,33 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [9.0.0]
+### Added
+- **BREAKING**: Add required `getSelectedAddress` callback argument to constructor ([#1579](https://github.com/MetaMask/core/pull/1579))
+- **BREAKING**: Add `isSupportedNetwork` method to `RemoteTransactionSource` interface ([#1579](https://github.com/MetaMask/core/pull/1579))
+- Add `baseFeePerGas` to transaction metadata ([#1590](https://github.com/MetaMask/core/pull/1590))
+- Add `txReceipt` to transaction metadata ([#1592](https://github.com/MetaMask/core/pull/1592))
+- Add `initApprovals` method to generate approval requests from unapproved transactions ([#1575](https://github.com/MetaMask/core/pull/1575))
+- Add optional `incomingTransactions` constructor arguments ([#1579](https://github.com/MetaMask/core/pull/1579))
+  - `apiKey`
+  - `includeTokenTransfers`
+  - `isEnabled`
+  - `updateTransactions`
+- Add incoming transaction methods ([#1579](https://github.com/MetaMask/core/pull/1579))
+  - `startIncomingTransactionPolling`
+  - `stopIncomingTransactionPolling`
+  - `updateIncomingTransactions` 
+- Add `requireApproval` option to `addTransaction` method options ([#1580](https://github.com/MetaMask/core/pull/1580))
+- Add `address` argument to `wipeTransactions` method ([#1573](https://github.com/MetaMask/core/pull/1573))
+
+### Changed
+- **BREAKING**: Move all but first argument to options bag in `addTransaction` method ([#1576](https://github.com/MetaMask/core/pull/1576))
+- **BREAKING**: Update `fromBlock` and `limit` types in `RemoteTransactionSourceRequest` ([#1579](https://github.com/MetaMask/core/pull/1579))
```

Comment:
Nit: Reading this from the perspective of a mobile/extension maintainer, it's unclear how these have changed, or what actions I should take to accommodate these changes. If the "recommended action" for a breaking change isn't obvious, additional detail is greatly appreciated.

In this case it seems kinda moot, because it's a type that seems primarily meant for internal use. But from reading the changelog, it seemed like a change I'd need to handle at first.

Here's an attempt to explain that better, and also it corrects a mistake regarding the `limit` property (`limit` was unaffected, it was `networkType` that was removed)

```suggestion
- **BREAKING**: Update properties of `RemoteTransactionSourceRequest` type ([#1579](https://github.com/MetaMask/core/pull/1579))
  - The `fromBlock` property has changed from `string` to `number`
  - The `networkType` property has been removed
  - This type is intended mainly for internal use, so it's likely this change doesn't affect most projects
```

Reply:
I wasn't sure how verbose we should be in these changelogs, but this is indeed much clearer, done.